runtime/src/snapshot_utils.rs

```diff
@@ -1179,9 +1179,7 @@ fn spawn_unpack_snapshot_thread(
     thread_index: usize,
 ) -> JoinHandle<()> {
     Builder::new()
-        .name(format!(
-            "solana-streaming-unarchive-snapshot-{thread_index}"
-        ))
+        .name(format!("solUnpkSnpsh-{thread_index:02}"))
```

Comment:
nit: imo drop the `-`, it doesn't add anything and gives another character to be used elsewhere if you want